Auth.py

```diff
@@ -28,7 +28,14 @@ def __init__(self, servername, host, username, password, database):
         self.username = username
         self.password = password
         self.database = database
-
+        
+        #Run the tests
```

Comment:
If you want, you can create a `run_tests` function which will run all of these tests.